ti/instrumentation/utils.go

```diff
@@ -196,31 +197,47 @@ func createConfigFile(runner TestRunner, packages, annotations, workspace, tmpDi
 		}
 		packages = strings.Join(pkgs, ",")
 	}
+	var data string
+	var outputFile string
 
-	data := fmt.Sprintf(`outDir: %s
+	if !yaml {
+		outputFile = fmt.Sprintf("%s/config.ini", tmpDir)
+		data = fmt.Sprintf(`outDir: %s
 logLevel: 0
 logConsole: false
 writeTo: COVERAGE_JSON
 instrPackages: %s`, dir, packages)
+	} else {
+		outputFile = fmt.Sprintf("%s/Config.yaml", tmpDir)
```

Comment:
Shouldn't file name be small case config.yml? Yaml file extension is yml.

Reply:
with the new changes to dotnet agent, the filename can be anything. @vistaarjuneja pls change accordingly